samples/bidirectional-chat/README.md

```diff
@@ -94,14 +94,27 @@ It's a quick try of this sample. You will create an Azure SignalR Service and an
 
 1. Enable function app cross origin resource sharing (CORS)
 
-    Although there is a CORS setting in local.settings.json, it is not propagated to the function app in Azure. You need to set it separately.
-
-    1. Open the function app in the Azure Portal.
-    2. In the left blade, select **CORS** blade.
-    3. In the **Allowed Origins** section, add `http://127.0.0.1:8080` (It is the local web server's url).
-    4. In order for the SignalR JavaScript SDK call your function app from a browser, support for credentials in CORS must be enabled. Select the **Enable Access-Control-Allow-Credentials** checkbox.
-    5. Click **Save** to persist the CORS settings.
-    ![CORS](content/cors.png)
+    * On local, in the `Host` section in the *local.settings.json* file, add `CORS` and `CORSCredentials` properties.
+        ```json
+            {
+                "IsEncrypted": false,
+                "Values": {
+                    // values
+                },
+                "Host": {
+                    "CORS": "http://localhost:8080",
```

Comment:
Use `127.0.0.1` to keep consistent with sample.json below?

Reply:
fixed